### ISSUE TITLE
Python 3.9.0 - 'Thread' object has no attribute 'isAlive'

### ISSUE THREAD

**Issue by Timothy-Dement** [IBM/ibm-cos-sdk-python#41]
# Versions

```
> $ sw_vers

ProductName:	Mac OS X
ProductVersion:	10.15.7
BuildVersion:	19H2
```
```
> $ python --version

Python 3.9.0
```
```
> $ pip show ibm-cos-sdk

Name: ibm-cos-sdk
Version: 2.8.0
Summary: IBM SDK for Python
Home-page: https://github.com/ibm/ibm-cos-sdk-python
Author: IBM
Author-email: None
License: Apache License 2.0
Location: /Users/tmdement/.pyenv/versions/3.9.0/lib/python3.9/site-packages
Requires: ibm-cos-sdk-core, ibm-cos-sdk-s3transfer, jmespath
Required-by: 
```

# Error

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/Users/tmdement/.pyenv/versions/3.9.0/lib/python3.9/site-packages/ibm_botocore/credentials.py", line 2227, in _cleanup
    self.stop_refresh_thread()
  File "/Users/tmdement/.pyenv/versions/3.9.0/lib/python3.9/site-packages/ibm_botocore/credentials.py", line 2236, in stop_refresh_thread
    if self._background_thread.isAlive():
AttributeError: 'Thread' object has no attribute 'isAlive'
```

# Likely Cause

It seems that in `Python 3.9.0` the `isAlive` method was renamed `is_alive`:

- https://github.com/Azure/azure-cli/issues/15441#issuecomment-705400107
- https://github.com/micheles/decorator/issues/75

**Comment by IBMeric** [comment]
Thank you for your comment. We are aware of this change, which we are targeting for our next release.

This is a duplicate of https://github.com/IBM/ibm-cos-sdk-python-core/issues/10.

**Comment by IBMeric** [comment]
This has been resolved in the 2.9.0 release. Please verify and then close this ticket.
